pkg/module/gitsource/git_test.go

```diff
@@ -18,12 +18,20 @@ func Test_fetchRepoURLFromRemotes(t *testing.T) {
 		want    string
 		wantErr bool
 	}{
-		{name: "Should return correct GitHub URL",
+		{name: "Should return correct GitHub URL - Git default format",
 			args: args{
 				gitRemotes: createTestRemotes(),
-				remoteName: "upstream",
+				remoteName: "upstream-git-default",
+			},
+			want:    "https://github.com/kyma-test/test.git",
+			wantErr: false,
+		},
+		{name: "Should return correct GitHub URL -  https format",
+			args: args{
+				gitRemotes: createTestRemotes(),
+				remoteName: "upstream-https",
 			},
-			want:    "github.com/kyma-test/test",
+			want:    "https://github.com/kyma-test/test.git",
 			wantErr: false,
 		},
```

Comment:
It would be useful to show what happens if there's a string without scheme prefix at all, like in the linked issue, where they used: `github.com/kyma-project/kyma-module`
There is a test repository in the `git_test.go` that uses such pseudo-URL, called `origin`, but there's no test case for it...
How about adding one? It would look like this:
```
        {name: "Should return unchanged string -  no scheme format",
                args: args{
                        gitRemotes: createTestRemotes(),
                        remoteName: "origin",
                },
                want:    "github.com/user-test/test",
                wantErr: false,
        },
```
What do you think?

Reply:
Will add it